katalog/gatekeeper/gpm/MAINTENANCE.md

```diff
@@ -0,0 +1,10 @@
+# Gatekeeper Policy Manager Package Maintenance
+
+To maintain the GPM package, you'll need to keep updated with upstream the following files:
```

Comment:
Let's add the link for the upstream project somewhere

Reply:
it is as a comment in each YAML file (it is written a little below at line 10) and the upstream project is linked in the readme also. Is this what you mean or something else?